client/galaxy/scripts/components/Workflow/Run/WorkflowRunSuccess.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+    <div>
+        <div class="donemessagelarge">
+            <p>
+                Successfully invoked workflow <b>{{ workflowName }}</b
+                ><em v-if="multipleInvocations"> - {{ timesExecuted }} times</em>.
+            </p>
+            <p v-if="multipleInvocations">
+                This workflow will generate results in multiple histories. You can observe progress in the
+                <a :href="historyTarget">history multi-view</a>.
+            </p>
+            <p v-else-if="wasNewHistoryTarget">
+                This workflow will generate results in a new history.
+                <a :href="historyTarget">Switch to that history now</a>.
+            </p>
+            <p v-else>
+                You can check the status of queued jobs and view the resulting data by refreshing the History pane, if
+                this has not already happened automatically.
+            </p>
+        </div>
+        <workflow-invocation-state
+            v-for="invocation in invocations"
+            v-bind:key="invocation.id"
```

Comment:
Is equal to :key, moving forward we should decide if we want to omit v-bind or not.

Reply:
I'd vote for standardizing to the shorthand syntax everywhere.  `:key` here, `@click` for click actions, and so on.